internal/config/config.go

```diff
@@ -43,49 +44,116 @@ type Jobs struct {
 }
 
 type Scenarios struct {
-	HighVolumeWrites HighVolumeWrites `yaml:"highVolumeWrites"`
-	HighVolumeReads  HighVolumeReads  `yaml:"highVolumeReads"`
+	HighVolumeWrites *HighVolumeWrites `yaml:"highVolumeWrites,omitempty"`
+	HighVolumeReads  *HighVolumeReads  `yaml:"highVolumeReads,omitempty"`
```

Comment:
Since we are down to two test suites that are used via parameters to tests different writes-only or reads-only test, why not finally rename then to get rid off the old `HighVolume` prefix? It used to be useful when the test case code was reflecting one case namely the `HighVolume` case. I would rename these and the files to: `IngestionPath` and `QueryPath`.